dashboard/dashboard.html

```diff
@@ -109,17 +109,16 @@
       <figure id="amherst_covid_case_data">
         <figcaption>
           <ul role="list">
-            <li role="listitem"><dfn>Vaccine Phase</dfn><span id="stat_phase"></span><a href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases">Vaccine info</a></li>
-            <li role="listitem"><dfn>COVID Status</dfn><span id="stat_status"></span><a href="https://county.pueblo.org/public-health/covid-19">Code info</a></li>
+            <li role="listitem"><dfn>Vaccine Phase</dfn><span id="stat_phase"></span><a href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases">Vaccine phases info</a></li>
+            <li role="listitem"><dfn>Is Amherst Clinic Open?</dfn><span id="stat_status"></span><a href="https://www.amherstcovid19.org/vaccine">Vaccine appointments</a></li>
```

Comment:
I thought that the yes/no here was supposed to reflect whether there are vaccine appointments available, but I may have misheard. We should check in with Brianna about this.

Reply:
Let's double check. My understanding was it was whether Amherst clinics are running or not and it would link to clinics to make appointments, but not be pulling appointment availability directly.